plugins/flatpak/gs-flatpak.c

```diff
@@ -214,6 +215,46 @@ remote_is_eos_apps (FlatpakRemote *remote)
 }
 
 static gboolean
+gs_flatpak_is_removable_remote (GsFlatpak *self,
```

Comment:
Thanks. It seemed like a lot of work for what I needed but I failed to find a better solution. I will give your suggestion a try.